src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ CHANGELOG
  * Attach the workflow's configuration to the `workflow` tag
  * Add the `allowed_recipients` option for mailer to allow some users to receive
    emails even if `recipients` is defined.
+ * Add support for setting `headers` with `Symfony\Bundle\FrameworkBundle\Controller\TemplateController`
```

Comment:
Please add a 7.2 headline, we are in feature freeze for 7.1

Reply:
Done